MAINTAINERS.md

```diff
@@ -84,3 +84,177 @@ Submit a PR with the changes and request a core team review.
+* Ansible core configuration is in the `ansible-documentation` repository at [https://github.com/ansible/ansible-documentation/blob/devel/.readthedocs.yaml](https://github.com/ansible/ansible-documentation/blob/devel/.readthedocs.yaml)
+
+> Each branch has its own Read the Docs configuration file.
+
+Most of the time there is no need to update the Read the Docs configuration files.
+The only change that should be necessary between versions is the Python version, for example:
+
+```yaml
+build:
+  os: ubuntu-22.04
+  tools:
+    python: >-
+      3.11
+```
+
+This Python version should match whatever version is used in the `ansible-documentation` repository to compile requirements and build the documentation.
+For example, the `stable-2.18` branch uses Python 3.11.
+The Python version should be also be 3.11 in the Read the Docs configuration for both Ansible package and Ansible core.
+
+> In the Read the Docs settings, the **Build pull requests for this project** option is selected only for the Ansible core project.
+> The configuration file for that project is in the `ansible-documentation` repository, which accepts pull requests.
+> The `package-doc-builds` repository is used to hold generated artifacts from the build workflow and does not get pull requests.
+
+### Updating Read the Docs for new releases
+
+When there is a new version of the Ansible package or Ansible core, someone needs to updates the Read the Docs project.
+
+To update the Ansible package project, do the following:
+
+1. Ensure you have maintainer access to the project.
+2. Navigate to the [package-doc-builds dashboard](https://app.readthedocs.org/projects/package-doc-builds/).
+3. Select the **Add version** button and select the new version to activate, which should be a new `XX` branch.
+4. Wait for the new version build to complete and then edit the main project settings.
+5. From the **Default branch** drop-down, select the new version that should correspond to "latest" and save your changes.
+
+To update the Ansible core project, do the following:
+
+1. Ensure you have maintainer access to the project.
+2. Navigate to the [ansible-core dashboard](https://app.readthedocs.org/projects/ansible-core/).
+3. Select the **Add version** button and select the new version to activate, which should be a new `stable-*` branch.
+
+### Hiding versions on Read the Docs
+
+Hiding older versions removes them from the fly-out menu on Read the Docs as well as search results.
+Older versions of the documentation that are not supported or EOL should be hidden.
+It is also possible to hide doc builds before they are released to evaluate the content.
+At release day, all that is needed is to toggle the switch so the build is no longer hidden and, in the case of the package docs, update the default branch for the project.
```

Comment:
So once Ansible 9 EOLs, the version flyout for package docs should only have latest and devel.

Reply:
TBH I'm not sure how that comment is related to this section. However, I'm afraid that's not quite the case. The version flyout will show `latest|devel|10|11` and any other active versions.